inc/Addon/Cloudflare/Subscriber.php

```diff
@@ -440,25 +440,25 @@ private function save_cloudflare_auto_settings( $auto_settings, $old_settings )
 	 * @param array $old_value An array of previous values for the settings.
 	 * @param array $value     An array of submitted values for the settings.
 	 *
-	 * @return void
+	 * @return array
```

Comment:
`update_option_` is an action, callback method doesn't need any returned value, why this change?